src/tools/profile/read_profile.c

```diff
@@ -216,54 +371,59 @@ static int show_profile_data_accum(profile_data_t *data, options_t *opts)
             continue;
         }
 
-        loc = &data->locations[sorted_loc->location_idx];
+        loc             = &data->locations[sorted_loc->location_idx];
+        loc_thread_info = &locations_thread_info[sorted_loc->location_idx];
 
         switch (loc->type) {
-        case UCS_PROFILE_TYPE_SAMPLE:
-            printf("%*.*s %13s %13s %10zu %18s:%-4d %s()\n",
-                   FUNC_NAME_MAX_LEN, FUNC_NAME_MAX_LEN,
-                   loc->name,
-                   "-",
-                   "-",
-                   sorted_loc->count,
-                   loc->file, loc->line, loc->function);
-            break;
         case UCS_PROFILE_TYPE_SCOPE_END:
-            printf("%*.*s %13.3f %13.0f %10zu %18s:%-4d %s()\n",
-                   FUNC_NAME_MAX_LEN, FUNC_NAME_MAX_LEN,
-                   loc->name,
-                   time_to_usec(data, opts, sorted_loc->total_time) /
-                   sorted_loc->count,
-                   time_to_usec(data, opts, sorted_loc->total_time),
-                   sorted_loc->count,
-                   loc->file, loc->line, loc->function);
+            snprintf(avg_buf,     sizeof(avg_buf) - 1, "%.3f",
```

Comment:
minor: something wrong with indent of ```sizeof```

Reply:
it's aligned with the sizeof() below (lines 382 and 384)